src/server/auth.ts

```diff
@@ -82,3 +84,13 @@ export const getServerAuthSession = (ctx: {
 }) => {
   return getServerSession(ctx.req, ctx.res, authOptions);
 };
+
```

Comment:
I feel like `mockSession` should belong elsewhere because I don't know if mock sessions ever get created in a normal/non-test usage scenario